src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -803,6 +884,63 @@ components:
         - photoLocation
         - validOnDate
 
+    AnonymousElectorDocumentSummariesResponse:
+      title: AnonymousElectorDocumentSummariesResponse
+      type: object
+      description: A response containing a collection of Anonymous Elector Document summaries
```

Comment:
Will the most recent one be returned first?

Reply:
Let's say we will do that - description updated 👍